.autorc

```diff
@@ -4,7 +4,6 @@
     "email": "eric@deployfx.com"
   },
   "plugins": [
-    ["docker", { "registry": "ejhayes/nodejs-bull-monitor", "tagLatest": false}],
```

Comment:
I wanted to have the build-push-action export to docker locally and keep the auto shipit action responsible for pushing to docker, but ran into this lmitation:

> The default image store in Docker Engine doesn't support loading multi-platform images. You can enable the containerd image store, or push multi-platform images is to directly push to a registry, see [registry](https://docs.docker.com/engine/reference/commandline/buildx_build/#registry).

https://docs.docker.com/engine/reference/commandline/buildx_build/#docker